sdk/storage/storage-blob-changefeed/tsconfig.src.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../../tsconfig.lib.json",
+  "compilerOptions": {
+    "lib": [
+      "DOM"
```

Comment:
Is this needed? The library seems to heavily rely on nodejs APIs and I doubt that it works in the browser.

Reply:
Yes, there are things that are in the DOM library that are required and are in the existing TSConfig